src/main/webapp/app/course/dashboards/assessment-dashboard/assessment-dashboard.component.ts

```diff
@@ -107,6 +108,7 @@ export class AssessmentDashboardComponent implements OnInit, AfterViewInit {
                 this.exam = res.body!;
                 this.course = Course.from(this.exam.course!);
                 this.courseService.checkAndSetCourseRights(this.course);
+                this.isAtLeastInstructor = this.accountService.isAtLeastInstructorInCourse(this.course);
```

Comment:
You need to do the same in the `else` branch. I don't think we need the button in non-exam mode as well.

Reply:
Thanks, I added it.